src/execution/operator/helper/physical_load.cpp

```diff
@@ -6,11 +6,18 @@
 #ifndef _WIN32
 #include <dlfcn.h>
 #else
+#ifndef NOMINMAX
```

Comment:
Since this seems to be repeated from `file_system.cpp`, could we perhaps extract this into `common/windows.hpp`?

Reply:
Sure. Take a look at the updated commit d97ad8a. Let me know if that was what you're thinking.

There's also `common/windows_undef.hpp` that seems to manually undefine min/max and some others. Additionally, the `windows_ci.py` adds an `#include <windows.h>` to `common/common.hpp`. 

Perhaps these are not needed anymore?

Reply:
windows_ci.py is used to test the windows include in different places, so that should stay. Windows_undefs should indeed perhaps be merged with windows.hpp.

Reply:
Taking a look at where `windows_undef.hpp` is included, it appears in places where `windows.h` is not included so seems to serve a different purpose that `windows.hpp`?

Apologies, as I'm not as familiar with the C++ code, so I'm less comfortable merging `windows_undef.hpp` into `windows.hpp` from what I'm seeing.

Reply:
Ah yeah, after having a look at this again indeed this should be kept separate. The `windows_undefs.hpp` is used to avoid conflicts caused by windows defines that are otherwise unavoidable (e.g. collision between `min` and `std::numeric_limits<int8_t>::max()`). It is fine to leave both as is then.